fahrschein/src/main/java/org/zalando/fahrschein/domain/Metadata.java

```diff
@@ -18,24 +16,29 @@
     private final OffsetDateTime occurredAt;
     private final OffsetDateTime receivedAt;
     private final String flowId;
-    private final Map<String, Object> other = new HashMap<String, Object>();
+    private final Map<String, String> spanCtx;
 
     @JsonCreator
     @Deprecated
-    private Metadata(@JsonProperty("event_type") String eventType, @JsonProperty("eid") String eid, @JsonProperty("occurred_at") String occurredAt, @JsonProperty("received_at") String receivedAt, @JsonProperty("flow_id") String flowId) {
-        this(eventType, eid, occurredAt == null ? null : OffsetDateTime.parse(occurredAt), receivedAt == null ? null : OffsetDateTime.parse(receivedAt), flowId);
+    private Metadata(@JsonProperty("event_type") String eventType, @JsonProperty("eid") String eid, @JsonProperty("occurred_at") String occurredAt, @JsonProperty("received_at") String receivedAt, @JsonProperty("flow_id") String flowId, @JsonProperty("span_ctx") Map<String, String> spanCtx) {
```

Comment:
Does it need to be "span_ctx"? Why not "span_context"?

Reply:
Good point. On the other hand we already have `eid` instead of event id, so everything is directly matching the names from the nakadi api.

Reply:
Is the property name on nakadi side already decided? @pc-alves

Reply:
Yes, the change was done using `span_ctx` and not likely to be changed any time soon.